ecs.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+use PhpCsFixer\Fixer\ArrayNotation\ArraySyntaxFixer;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symplify\EasyCodingStandard\ValueObject\Option;
+use Symplify\EasyCodingStandard\ValueObject\Set\SetList;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+    $services->set(ArraySyntaxFixer::class)
+        ->call('configure', [[
+            'syntax' => 'short',
+        ]]);
+
+    $parameters = $containerConfigurator->parameters();
+    $parameters->set(Option::PATHS, [
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+        __DIR__ . '/ecs.php',
+    ]);
+
+    $parameters->set(Option::SETS, [
+        SetList::SPACES,
+        // run and fix, one by one
+        // SetList::PSR_12,
+        // SetList::PHP_70,
+        // SetList::PHP_71,
+    ]);
```

Comment:
The goal is to eventually add all the common sets, moreover PSR-12. 
But we should start with something simple and safe as "spaces" set.

The next sets will follow in next PRs after this get merged, to keep it simple.